### ISSUE TITLE
Transfer Learning wih Cityscapes Classes

### ISSUE THREAD

**Issue by CptK1ng** [matterport/Mask_RCNN#774]
Hi, sorry that i post kind of offtopic stuff here, but i have a few questions about the topology of the Mask R-CNN.

I want to use the pretrained model for transfer learning with a dataset based on cityscapes classes. To decide which of the layers i need to freeze and which one to train again with new dataset, i need some informations about the ResNet 101. Whats the easiest way to get the architecture of the network and are there any suggestions which layers i should replace with new one. 

Dataset has 25000 Pictures with ground truth. https://download.visinf.tu-darmstadt.de/data/from_games/

I would be very appreciate for your help. 
Thanks

**Comment by nivedyaammu** [comment]
> Hi, sorry that i post kind of offtopic stuff here, but i have a few questions about the topology of the Mask R-CNN.
> 
> I want to use the pretrained model for transfer learning with a dataset based on cityscapes classes. To decide which of the layers i need to freeze and which one to train again with new dataset, i need some informations about the ResNet 101. Whats the easiest way to get the architecture of the network and are there any suggestions which layers i should replace with new one.
> 
> Dataset has 25000 Pictures with ground truth. https://download.visinf.tu-darmstadt.de/data/from_games/
> 
> I would be very appreciate for your help.
> Thanks

How did you modfiiy your cityscape data to match mask-rcnn. Can you help?
